besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java

```diff
@@ -1905,6 +1916,24 @@ private GraphQLConfiguration graphQLConfiguration() {
     return graphQLConfiguration;
   }
 
+  private JsonRpcConfiguration engineJsonRpcConfiguration() {
+    final JsonRpcConfiguration engineConfig = JsonRpcConfiguration.createDefault();
+    engineConfig.setEnabled(isRpcHttpEnabled);
+    engineConfig.setHost(rpcHttpHost);
+    engineConfig.setPort(engineRpcHttpPort);
+    engineConfig.setMaxActiveConnections(rpcHttpMaxConnections);
+    engineConfig.setCorsAllowedDomains(rpcHttpCorsAllowedOrigins);
+    engineConfig.setRpcApis(rpcHttpApis.stream().distinct().collect(Collectors.toList()));
```

Comment:
I think execution only needs ENGINE and ETH (and maybe WEB3).  I think it might be safer to be explicit about what APIs are on the execution endpoint rather than rely on CLI params

Reply:
So now i'm thinking we should add config opts for `engineHostsAllowList` for folks to control where CLs can/cannot connect from. What about the max connections and CORS settings too, should we have engine specific options for those instead of inheriting from the broader RPC settings?